src/style.css

```
@@ -0,0 +1,38 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+}
+
+.stars--1 > :nth-child(1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 > :nth-child(-n+2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 > :nth-child(-n+3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 > :nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 > :nth-child(n) {
+  background-image: url(./images/star-active.svg);
+}
```

```suggestion
.stars--1 :nth-child(1) {
  background-image: url(./images/star-active.svg);
}

.stars--2 :nth-child(-n+2) {
  background-image: url(./images/star-active.svg);
}

.stars--3 :nth-child(-n+3) {
  background-image: url(./images/star-active.svg);
}

.stars--4 :nth-child(-n+4) {
  background-image: url(./images/star-active.svg);
}

.stars--5 :nth-child(n) {
  background-image: url(./images/star-active.svg);
}
```
Запиши эти правила через запятую, они применяют один и тот же стиль

---

сделал